### PR TITLE
Develop Random SSN Generator

### PR DESCRIPTION
**Pull request by luiz-arun**
### What is the changes

Develop TestProject-Addons for UI automation. This PR is to set up the infra and develop a random ssn generator.

### CODE REVIEW

**Comment by zhlim-ml** [comment]
general .idea folder should be ignored.

File: src/main/resources/manifest.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"guid":"FUspeHguZEO1acdrBXhMjQ","version":"1.0","permissions":[]}
```
Comment (zhlim-ml) [inline src/main/resources/manifest.json]: I guess the guid maybe sensitive for deployment i guess please leave it to local only.

**Review by zhlim-ml — CHANGES_REQUESTED** [review]
add .idea folder into ignore list.
add manifest.json into ignore list.

File: testproject-addon.iml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module org.jetbrains.idea.maven.project.MavenProjectsManager.isMavenModule="true" type="JAVA_MODULE" version="4">
```
Comment (zhlim-ml) [inline testproject-addon.iml]: this should be ignore as well

**Comment by luiz-arun** [comment]
All done @zhlim-ml

**Review by zhlim-ml — APPROVED** [review]
LGTM

**Pull request merged**